lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/ShowPageProducer.java

```diff
@@ -5314,4 +5320,16 @@ private void makeSamplePeerEval(UIContainer parent)
 		UIOutput.make(peerReviewRows, "peer-eval-sample-text", messageLocator.getMessage("simplepage.peer-eval.sample.4"));
 	}
 
+	private String personalizeText(String itemText) {
+		if (StringUtils.isNotBlank(itemText)) {
+			User currentUser = UserDirectoryService.getCurrentUser();
+			if (currentUser != null) {
```

Comment:
Yeah, although if it can't get an instance of the `UserDirectoryService` it's just not worth carrying on and everything will actually have blown up allot earlier.

In templating languages often the template will specify the default eg: `{{lastname|Unknown}}` but this might be overcomplicating what should be a simple issue. I'd be happy with a blank.